be/src/agent/agent_server.cpp

```diff
@@ -144,7 +168,7 @@ void AgentServer::Impl::init_or_die() {
     CREATE_AND_START_POOL(DELETE, _delete_workers,
                           config::delete_worker_count_normal_priority + config::delete_worker_count_high_priority);
     CREATE_AND_START_POOL(ALTER_TABLE, _alter_tablet_workers, config::alter_tablet_worker_count);
-    CREATE_AND_START_POOL(CLONE, _clone_workers, config::clone_worker_count);
+    CREATE_AND_START_POOL(CLONE, _clone_workers, 1);
```

Comment:
This clone pool can be removed or not?

Reply:
cannot remove, because we need this worker pool to handle task to dynamic pool, why this is necessary, you can see the comments above.